save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -0,0 +1,71 @@
+package com.saveourtool.save.backend.controllers
+
+import com.saveourtool.save.backend.configs.ConfigProperties
+import com.saveourtool.save.configs.RequiresAuthorizationSourceHeader
+import com.saveourtool.save.demo.NewDemoToolRequest
+import com.saveourtool.save.spring.utils.applyAll
+import com.saveourtool.save.utils.EmptyResponse
+import com.saveourtool.save.v1
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.Parameters
+import io.swagger.v3.oas.annotations.enums.ParameterIn
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.springframework.boot.web.reactive.function.client.WebClientCustomizer
+import org.springframework.http.MediaType
+import org.springframework.http.codec.json.Jackson2JsonEncoder
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+
+/**
+ * Controller that allows adding tools to save-demo
+ */
+@RestController
+@RequestMapping("/api/$v1/demo")
+class DemoManagerController(
+    configProperties: ConfigProperties,
+    objectMapper: ObjectMapper,
+    customizers: List<WebClientCustomizer>,
+) {
+    private val webClientDemo = WebClient.builder()
+        .baseUrl(configProperties.demoUrl)
+        .codecs {
+            it.defaultCodecs().multipartCodecs().encoder(Jackson2JsonEncoder(objectMapper))
```

Comment:
do we need it?
as I remember, such customizations were moved to  `customizers: List<WebClientCustomizer>`. If we really need codec for multipart, it can be moved to `com.saveourtool.save.backend.configs.WebConfig#jackson2WebClientCustomizer`

Reply:
I guess for now it is not needed, but if we plan to pass archives between backend and demo, it might be nice

Reply:
Removed it